proto/osmosis/gamm/v1beta1/tx.proto

```diff
@@ -75,59 +71,6 @@ message MsgExitPoolResponse {
   ];
 }
 
-// ===================== MsgSwapExactAmountIn
-message SwapAmountInRoute {
-  uint64 pool_id = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
-  string token_out_denom = 2
-      [ (gogoproto.moretags) = "yaml:\"token_out_denom\"" ];
-}
```

Comment:
We only need to remove this.

Please do not remove any messages or queries. We need to keep those for backward compatibility for clients.

Replace `SwapAmountInRoute` in gamm with `SwapAmountInRoute` in `poolmanager`

Let me know if anything doesn't make sense

Reply:
Got it